assets/stylesheets/sections/post-editor.scss

```diff
@@ -7,6 +7,7 @@
 @import '../main';
 @import '../shared/forms';
 @import 'components/accordion/style';
+@import 'components/card/style';
```

Comment:
I'm not entirely sure why we need that import now, but without it I get the following error:

```
{
  "formatted": "Error: \".editor-confirmation-sidebar__ground-control\" failed to @extend \".card\".\n       The selector \".card\" was not found.\n       Use \"@extend .card !optional\" if the extend should be able to fail.\n        on line 73 of client/post-editor/editor-confirmation-sidebar/style.scss\n>> \t@extend .card;\n   -^\n",
  "message": "\".editor-confirmation-sidebar__ground-control\" failed to @extend \".card\".\nThe selector \".card\" was not found.\nUse \"@extend .card !optional\" if the extend should be able to fail.",
  "column": 2,
  "line": 73,
  "file": "client/post-editor/editor-confirmation-sidebar/style.scss",
  "status": 1
}
```

Reply:
Maybe previously there was something else that was importing that so it ended up ok

Reply:
Note that after this PR was merged, the editor confirmation sidebar ground control styling was broken. GH issue and fix for that coming soon -- not sure why that affected things as such, but I think I can fix up the editor confirmation sidebar ground control styling to be a bit more robust.

![pasted_image_at_2017_10_03_11_19_am](https://user-images.githubusercontent.com/2098816/31141260-e331349a-a844-11e7-8e5d-d3921232ea8c.png)